### PR TITLE
Updated Python Docs

### PR DESCRIPTION
**Pull request by Dorthu**
Changed dnszone to domian, added nodebalancer.yaml, renamed methods
in linode_client to match current deployment.

### CODE REVIEW

File: docs/src/data/python/domain.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            Sends any local changes to Editable fields of this Domain to the API, saving them.
+        example: |
+            domain.save()
+        returns: True if the save succeeded
+    delete:
+        desc: >
+            Deletes this Domain from your account.  This is not reversable.
+        example:
+            domain.delete()
+        returns: True if the delete succueeded.
```
Comment (mparke) [inline docs/src/data/python/domain.yaml]: typo

File: docs/src/data/python/domain.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        desc: Creates a new DomainRecord for this Domain
+        parameters:
+            record_type:
+                _keyword: false
+                desc: The type of record to create (A, AAAA, etc.)
+            '**kwargs':
+                _keyword: false
+                desc: >
+                    Any number of keyword arguments to send to the POST /domains/example.org endpoint.  See
+                    the <a href="/reference#ep-domains-records-POST">endpoint docs</a> for more information.
```
Comment (mparke) [inline docs/src/data/python/domain.yaml]: just making a note here, me and abe were debating how we should handle some of the explicit html in descriptions, right now it's all escaped. should we make exceptions and allow unescaped html for these circumstances? or some other approach

File: docs/src/data/python/linode_client.yaml [hunk 3]
```diff
@@ -27,6 +27,34 @@ methods:
...
+    get_domains:
+        desc: >
+            Retrieves a list of Domain objects.  This function is filterable - include filters in the
+            parameters list to retrieve only specific objects.
+        parameters:
+            '*filters':
+                desc: Any number of filters to this function
+                _keyword: false
+        example: >
+            domains = client.dns.get_zones()
```
Comment (mparke) [inline docs/src/data/python/linode_client.yaml]: domain? or still `dns`?

File: docs/src/data/python/nodebalancer.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            Sends any local changes to Editable fields of this Domain to the API, saving them.
+        example: |
+            nodebalancer.save()
+        returns: True if the save succeeded
+    delete:
+        desc: >
+            Deletes this NodeBalancer from your account.  This is not reversable.
+        example:
+            nodebalancer.delete()
+        returns: True if the delete succueeded.
```
Comment (mparke) [inline docs/src/data/python/nodebalancer.yaml]: same typo

File: docs/src/data/python/nodebalancer.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            label:
+                _keyword: true
+                desc: A label for this config
+            port:
+                _keyword: true
+                desc: The port for this NodeBalancerConfig
+            '**kwargs':
+                _keyword: false
+                desc: >
+                    Any number of keyword arguments to send to the POST /nodebalanacers/123/configs endpoint.  See
```
Comment (mparke) [inline docs/src/data/python/nodebalancer.yaml]: balanacers typo

File: docs/src/data/python/domain.yaml (hunk 2 above)
Comment (abemassry) [inline docs/src/data/python/domain.yaml]: We have a solution in a forthcoming PR

**Comment by na3d** [comment]
Needs put to develop instead of master.

File: docs/src/data/python/linode_client.yaml (hunk 3 above)
Comment (Dorthu) [inline docs/src/data/python/linode_client.yaml]: This should be client.get_domains()

**Pull request merged**